### PR TITLE
Fixing sequence header size and other UI bugs

### PR DESCRIPTION
**Pull request by asilver-d2l**
- Sequence header spans length of sidebar
- Fixed border issues when focusing on a link
- Fixed weird issue on Edge and Firefox ESR where the module-header was not getting scoped properly, so the element would not receive any styles (apparently adding a class fixed this).

### CODE REVIEW

File: components/d2l-inner-module.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ class D2LInnerModule extends ASVFocusWithinMixin(PolymerASVLaunchMixin(Completio
 		</style>
 
 		<div id="header-container" class$="[[isEmpty(subEntities)]]">
-			<div id="module-header" class$="[[_getIsSelected(currentActivity, entity, focusWithin)]]" on-click="_onHeaderClicked">
+			<div id="module-header" class$="[[_getIsSelected(currentActivity, entity, focusWithin)]] module-header" on-click="_onHeaderClicked">
```
Comment (benyamin-noori) [inline components/d2l-inner-module.js]: Why are we adding this here? There are no rules for the module-header class, only for module-header ID which this element gets anyway, right? Or am I missing something?

Comment (asilver-d2l) [reply components/d2l-inner-module.js]: It was a weird thing I experienced only on Firefox ESR. The element wouldn't get this `scoped to inner module` class (something like that). For some reason, when I added a class to the element, it received the other class, and styled correctly.

Comment (jwong-d2l) [reply components/d2l-inner-module.js]: do you have the exact class name?

Comment (asilver-d2l) [reply components/d2l-inner-module.js]: `style-scope d2l-inner-module`

Comment (asilver-d2l) [reply components/d2l-inner-module.js]: It seems to work fine if I just replace the class `module-header` with `style-scope d2l-inner-module`.

Comment (asilver-d2l) [reply components/d2l-inner-module.js]: Although on Firefox ESR, it has the class twice.

Comment (jwong-d2l) [reply components/d2l-inner-module.js]: weird, but kind of makes more sense?

Comment (asilver-d2l) [reply components/d2l-inner-module.js]: Do you want me to commit this change?

**Pull request merged**